offline/cable_LUC_EXPT.F90

```diff
@@ -929,38 +930,16 @@ SUBROUTINE READ_LUH2(LUC_EXPT)
        write(*,'(a)') 'warning: past end of LUH2 record'
     endif
 
+    ! Determine woody fraction (forest and shrub cover).
+    call get_woody_fraction(LUC_EXPT)
```

Comment:
Do we need to recalculate here - or can we just inherit %woodfrac from the call to get_woody_fraction from within LUC_EXPT_INIT?  

This going to end up rereading the Climate/MVG file every year so will slow things down.

If it does need to redone - then I suggest putting the read of MVG/Climate into LUC_EXPT_INIT and restrict get_woody_fraction() to go from %biome to %woody_fraction.

Reply:
Good point, I don't think it needs to be recalculated because LUC_EXPT_INIT is called before all occurrences of READ_LUH2 and because woodfrac does not change over time (maybe it should, but that's a different issue).

I deleted the two highlighted lines.